react/tic-tac-toe/src/app/screens/Home/screens/Matches/index.js

```diff
@@ -1,43 +1,18 @@
 import React, { useEffect } from 'react';
-import Spinner from 'react-spinkit';
-import { useSelector, useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 
 import gameActions from '~redux/game/actions';
 
-import styles from './styles.module.scss';
+import LayoutWithLoader from './layout';
```

Comment:
I'd name it just `Layout`. Although, what we usually do here at Wolox is:
`index.js => MatchesContainer`
`layout.js => Matches`